app/src/androidTest/kotlin/cz/covid19cz/erouska/screenObject/SMSScreen.kt

```diff
@@ -25,8 +25,22 @@ object SMSScreen {
     }
 
     fun verifyLater() {
-        await().ignoreExceptions().atMost(35, TimeUnit.SECONDS).untilAsserted {
-            onView(withId(R.id.login_verify_later_button)).click()
-        }
+
+        await().ignoreExceptions()
+            .atLeast(25, TimeUnit.SECONDS)
+            .atMost(35, TimeUnit.SECONDS)
+            .untilAsserted {
+                onView(withId(R.id.login_verify_later_button)).check(
+                    matches(
+                        not(
+                            ViewSizeMatcher(
```

Comment:
I had to change this assert, because the button is there with 0,0 size and Espresso was trying to click this button and was successful after like 2 seconds, also having here only atMost could miss issue that button is visible instantly

Reply:
Good, but add this as a comment to the code